### PR TITLE
[Feature] sandbox mode

### PR DESCRIPTION
**Pull request by rueshyna**
prepare sandbox of both tezos-node and tzkt's components
- docker-compose.stateless.yml: when containers is done, all the data will be gone
- docker-compose.stateful.yml: the data will remain.

### CODE REVIEW

**Review by JulesGuesnon — COMMENTED** [review]
The docker part looks good, but I think it lacks documentation. It would be nice to have explanations of the purpose and how to use all of this in the readme

**Comment by rueshyna** [comment]
updated readme

File: README.md [hunk 1]
```diff
@@ -30,8 +30,21 @@ Once you updated the code, you can build it with:
 npm run build
 ```
 
-And serve the build with:
+And server the build with:
```
Comment (JulesGuesnon) [inline README.md:33]: I think it's a typo, it should be `serve` as it was before

**Review by JulesGuesnon — APPROVED** [review]
Only a small typo, but otherwise LGTM

File: README.md (hunk 1 above)
Comment (rueshyna) [reply README.md]: modified.

**Pull request merged**